CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## [Carbonaceous-Comet(v29)](https://github.com/cyverse/atmosphere/milestone/16?closed=1) (as of 11/9/2017)
+## [Delightful-Duboshin (v30)](https://github.com/cyverse/atmosphere/milestone/17?closed=1) (as of 2017-12-18)
+
+New Features:
+-  Site operators can override enforcement behavior for specific allocation sources
+
+
+## [Carbonaceous-Comet (v29)](https://github.com/cyverse/atmosphere/milestone/16?closed=1) (as of 2017-11-09)
```

Comment:
I'd like you to know that Troposphere is now ISO 8601 compliant in it's [CHANGELOG.md](https://github.com/cyverse/troposphere/commit/cf56dbfc1f3774d8c19f720ff9dc05997346642a) too.